### PR TITLE
C100 dashboard tasklist changes

### PR DESCRIPTION
**Pull request by channa-patil**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `keep-helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by sahaSonali — APPROVED** [review]

File: definitions/private-law/json/CaseEvent/CaseEvent.json [hunk 1]
```diff
@@ -1690,5 +1690,17 @@
...
+  },
+  {
+    "LiveFrom": "01/03/2023",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "citizenCaseWithdraw",
+    "Name": "Withdraw Citizen Case",
+    "Description": "Withdraw a case submitted by citizen",
+    "PreConditionState(s)": "SUBMITTED_PAID;SUBMITTED_NOT_PAID;AWAITING_RESUBMISSION_TO_HMCTS;GATE_KEEPING;CASE_ISSUE",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/citizen-case-withdrawn-notification",
+    "PostConditionState": "CASE_WITHDRAWN",
```
Comment (AlokDatta) [inline definitions/private-law/json/CaseEvent/CaseEvent.json:1702]: Should this always be withdrawn? even when the case is in ISSUED/ Gatekeeping state?

File: definitions/private-law/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json [hunk 2]
```diff
@@ -128,5 +128,28 @@
...
+          "caseworker-privatelaw-systemupdate"
+        ],
+        "CRUD": "CRU"
+      },
+      {
+        "UserRoles": [
+          "caseworker-privatelaw-courtadmin",
+          "caseworker-privatelaw-judge",
+          "caseworker-privatelaw-la",
+          "caseworker-privatelaw-superuser"
```
Comment (AlokDatta) [inline definitions/private-law/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json:149]: Double checking, do we need these roles to have Read access, as probably they wont be served until when we are allowing withdrawal.

"[SOLICITORA]",
"[SOLICITORB]",
          "[SOLICITORC]",
          "[SOLICITORD]",
          "[SOLICITORE]",
          "[SOLICITORF]",
          "[SOLICITORG]",
          "[SOLICITORH]",
          "[SOLICITORI]",
          "[SOLICITORJ]",

File: definitions/private-law/json/CaseEvent/CaseEvent.json (hunk 1 above)
Comment (channa-patil) [reply definitions/private-law/json/CaseEvent/CaseEvent.json:1702]: Hi @AlokDatta, yes state will change to Withdrawn always as this event is only used in API when case is not issued yet(i.e in Submitted or Pending) state.

After the case is issued, withdrawal process is completely offline & this event is never invoked in those cases.
As you explained, withdrawal request status is always changed through an order to mark it as Closed.

File: definitions/private-law/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json (hunk 2 above)
Comment (channa-patil) [reply definitions/private-law/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json:149]: Hi @AlokDatta, No it's not required for solicitors as the event is exclusively available for citizens.

**Pull request merged**